client/src/Pages/CharacterPage.jsx

```diff
@@ -0,0 +1,163 @@
+import React from 'react'; 
+import {Typography, Divider} from 'antd'; 
+import jerome from "../Images/jerome.png";
+import Annalise from "../Images/DorsalFinGang/annalise-1.PNG";
+import Austin from "../Images/DorsalFinGang/austin-6.PNG";
+import Cat from "../Images/DorsalFinGang/cat-2.PNG";
+import Daniel from "../Images/DorsalFinGang/daniel-4.PNG";
+import Hannah from "../Images/DorsalFinGang/hannah-4.PNG";
+import Isaac from "../Images/DorsalFinGang/isaac-2.PNG";
+import Jacob from "../Images/DorsalFinGang/jacob-4.PNG";
+import Josh from "../Images/DorsalFinGang/josh-1.PNG";
+import Lucas from "../Images/DorsalFinGang/lucas-6.PNG";
+import Rachel from "../Images/DorsalFinGang/rachel-2.PNG";
+import Sydney from "../Images/DorsalFinGang/sydney-7.PNG";
+
+const {Title} = Typography; 
+
+
+export default class CharacterPage extends React.Component{
+
+    render(){
```

Comment:
There's a whole lot of repeat code here. What you could do instead is have an array of objects. where each one has a `image`, `title`, and `description` property. Then, map it like this:
```javascript
arr.map((role) => (
  <div>
    <Title level={4}>{role.title}</Title>
    <img src={role.image} alt={role.title} ... />
    <p ...>{role.description}</p>
    <Divider />
  </div>
));
```

Reply:
Since we are on a deadline. I'm going to leave this for now, then clean it up if we have time. I'll make an issue for it